app/src/main/resources/db/migration/ekstern_varsling_model/V9__notifikasjon_koordinat_sak.sql

```diff
@@ -0,0 +1,10 @@
+create table merkelapp_grupperingsid_notifikasjon
+(
+    notifikasjon_id uuid primary key not null,
+    merkelapp text not null,
+    grupperingsid text
+);
+
+create index merkelapp_grupperingsid_notifikasjon_merkelapp_idx on merkelapp_grupperingsid_notifikasjon (merkelapp);
+create index merkelapp_grupperingsid_notifikasjon_grupperingsid_idx on merkelapp_grupperingsid_notifikasjon (grupperingsid);
```

Comment:
Tror indeksen burde være på `(merkelapp, grupperingsid)` for spørringen 
```sql
select notifikasjon_id from merkelapp_grupperingsid_notifikasjon 
where merkelapp = ? and grupperingsid = ?
```

Reply:
6abcfc4